graphistry/feature_utils.py

```diff
@@ -91,6 +101,20 @@ def lazy_import_has_min_dependancy():
     except ModuleNotFoundError as e:
         return False, e
 
+def lazy_import_has_cuml_dependancy():
```

Comment:
rename `lazy_import_has_cucat_dependency()` ?

Reply:
sure, had just thought for future cudf or cuml all the way thru